pyppeteer/target.py

```diff
@@ -17,33 +19,46 @@
 class Target(object):
     """Browser's target class."""
 
-    def __init__(self, targetInfo: Dict, browserContext: 'BrowserContext',
-                 sessionFactory: Callable[[], Coroutine[Any, Any, CDPSession]],
-                 ignoreHTTPSErrors: bool, defaultViewport: Optional[Dict],
-                 screenshotTaskQueue: List, loop: asyncio.AbstractEventLoop
-                 ) -> None:
+    def __init__(
+            self,
+            targetInfo: Dict,
+            browserContext: 'BrowserContext',
+            sessionFactory: Callable[[], Coroutine[Any, Any, CDPSession]],
```

Comment:
Much simpler to read:
```suggestion
            sessionFactory: Callable[[], Awaitable[CDPSession]],
```
Would also require changing the imports.